webapp/package.json

```diff
@@ -7,11 +7,12 @@
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
     "bootstrap": "^5.2.0",
+    "chess.js": "^0.13.3",
```

Comment:
Incorrect version of Chess.js

https://trello.com/c/YBzCeCzg/27-back-%D0%B2%D1%8B%D0%B1%D1%80%D0%B0%D1%82%D1%8C-%D0%B2%D0%B5%D1%80%D1%81%D0%B8%D1%8E-%D0%B4%D0%BB%D1%8F-chessjs